src/main/webapp/app/exam/participate/exercises/text/text-exam-submission.component.scss

```
@@ -62,3 +62,21 @@ div {
         background-color: whitesmoke;
     }
 }
+
+#highlightDiffButton {
+    background-color: #fbc107;
+    border: none;
+    padding: 0px 10px;
```

to fix the cadacy issue you can remove the px after 0
```suggestion
    padding: 0 10px;
```